blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolder.java

```diff
@@ -0,0 +1,35 @@
+package io.jenkins.blueocean.rest.impl.pipeline;
+
+import hudson.model.Item;
+import io.jenkins.blueocean.rest.Reachable;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.service.embedded.rest.PipelineFolderImpl;
+import org.kohsuke.stapler.export.Exported;
+
+public abstract class OrganizationFolder extends PipelineFolderImpl {
```

Comment:
@i386 yeah, we can easily do that. I will pull out GitHubOrganizationFolder in to 573 and let you review. 573 doesn't deal with bitbucket yet, so that can go in another PR? I am hesitant putting these in pipeline-api-impl, so bitbucket specific org folder will go in separate bitbucket plugin when we do its creation story?